### PR TITLE
Introducing sql timeseries mapping source to PrimaryServiceProxy

### LINKED ISSUE #101: Implement SQL input for time series mapping
Analogous to #34, SQL input of time series mappings is needed in `PrimaryServiceProxy`.

### PR DESCRIPTION
**Pull request by sebastian-peter**
Resolves #101 

Merge first:
- #98

### CODE REVIEW

**Comment by sebastian-peter** [comment]
Ready for review. Found (somewhat) related issues #38 and #180, which can be tackled in the future

**Review by danielfeismann — DISMISSED** [review]
Looks good, thanks for the work :)

File: src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_p.sql [hunk 1]
```diff
@@ -10,7 +10,9 @@ CREATE TABLE public.time_series_p
 
 CREATE INDEX time_series_p_series_id ON time_series_p USING hash (time_series);
 
-CREATE UNIQUE INDEX time_series_p_series_time ON time_series_p USING  btree (time_series, time);
+-- Order of columns is important when using btree: https://www.postgresql.org/docs/14/indexes-multicolumn.html
+-- time_series at first since we at most use an equality constraint on time_series and a range query on time
```
Comment (t-ober) [inline src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_p.sql:14]: I don't quite get this sentence 👓

File: src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_pqh.sql [hunk 2]
```diff
@@ -12,7 +12,9 @@ CREATE TABLE public.time_series_pqh
 
 CREATE INDEX time_series_pqh_series_id ON time_series_pqh USING hash (time_series);
 
-CREATE UNIQUE INDEX time_series_pqh_series_time ON time_series_pqh USING  btree (time_series, time);
+-- Order of columns is important when using btree: https://www.postgresql.org/docs/14/indexes-multicolumn.html
+-- time_series at first since we at most use an equality constraint on time_series and a range query on time
```
Comment (t-ober) [inline src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_pqh.sql:16]: s.o.

File: src/test/scala/edu/ie3/simona/service/primary/PrimaryServiceProxySqlIT.scala [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+      )
+
+      val triggerIdInit1 = 1L
+
+      scheduler.send(
+        proxyRef,
+        TriggerWithIdMessage(
+          InitializeServiceTrigger(initData),
+          triggerIdInit1,
+          proxyRef
```
Comment (t-ober) [inline src/test/scala/edu/ie3/simona/service/primary/PrimaryServiceProxySqlIT.scala:118]: Just something I was wondering about: Do you know why we include the receiving party in the trigger that gets directly sent to that individual? Is this for debugging reasons ?

**Review by t-ober — CHANGES_REQUESTED** [review]
Looks great 🚀  Just a very minor thing !

**Comment by t-ober** [comment]
I think we can ignore the codacy method longer than 50 lines issue as this is a pattern matching statement which is perfectly understandable and readable

File: src/test/scala/edu/ie3/simona/service/primary/PrimaryServiceProxySqlIT.scala (hunk 3 above)
Comment (sebastian-peter) [reply src/test/scala/edu/ie3/simona/service/primary/PrimaryServiceProxySqlIT.scala:118]: No, currently it is not used anywhere. We can probably remove it, but I want to touch scheduling at some point in the future anyway.

File: src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_p.sql (hunk 1 above)
Comment (sebastian-peter) [reply src/test/resources/edu/ie3/simona/service/primary/timeseries/time_series_p.sql:14]: Well I didn't want to write a monograph here, but I try to explain this to you for sure

**Review by t-ober — APPROVED** [review]

**Review by danielfeismann — APPROVED** [review]
Looks good, thanks a lot :)

**Pull request merged**